test/parallel/test-webstream-readable-from.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+require('../common');
+const { throws } = require('node:assert');
+
+throws(() => ReadableStream.from({}), { code: 'ERR_ARG_NOT_ITERABLE', name: 'TypeError' });
```

Comment:
nit: it would be more readable to split it in several lines, and to use `assert.throws` instead of just `throws`
```suggestion
assert.throws(
  () => ReadableStream.from({}),
  { code: 'ERR_ARG_NOT_ITERABLE', name: 'TypeError' }
);
```

Reply:
Thanks for the suggestion, done.